Index: forum/links.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\nfrom urllib.parse import urlparse, urlsplit\n\ndef links(content):\n    url = urlparse(content)\n    if (url.scheme and url.netloc and url.path is not None) and len(url.netloc.split('.')) > 1:\n        url1 = ''\n        link_string = '<a href = \"'\n        link_string = link_string + content\n        link_string = link_string + '\">'\n        link_string = link_string + content + '</a><br>\\n'\n        url1 += link_string\n        return url1\n    else:\n        return content
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/forum/links.py b/forum/links.py
--- a/forum/links.py	
+++ b/forum/links.py	
@@ -1,4 +1,3 @@
-
 from urllib.parse import urlparse, urlsplit
 
 def links(content):
@@ -12,4 +11,4 @@
         url1 += link_string
         return url1
     else:
-        return content
\ No newline at end of file
+        return content
